configure/options.md

```diff
@@ -0,0 +1,39 @@
+Changing MySQL Options
+============================================================================
+
+MySQL allows the option to configure the database with a configuration file. You can pass the MySQL configuration options to the cluster in the following ways:
```

Comment:
```suggestion
You can pass MySQL options from the `my.cnf` configuration file to the cluster in the following ways:
```
Let's suppose the reader already knows about MySQL configuration file... Well, for those who doesn't know about it  we can make my.cnf a hyperlink to the upstream documentation page, e.g. https://dev.mysql.com/doc/refman/8.0/en/option-files.html